### PR TITLE
JETUT-72 catch up servlet and faces hello world as per changes in tutorial

### PR DESCRIPTION
**Pull request by BalusC**

### CODE REVIEW

**Comment by kito99** [comment]
@arjantijms please review

**Comment by kito99** [comment]
@mnriem can you review?

**Review by mnriem — CHANGES_REQUESTED** [review]

**Comment by kito99** [comment]
thanks @mnriem!

**Comment by mnriem** [comment]
@kito99 Do you need more from my end?

**Comment by kito99** [comment]
@mnriem I think we're good. thanks! 

@BalusC this is in your court, now.

File: tutorial/README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+- GlassFish 7
 
 ## Build the examples
 
-Unzip GlassFish on the predefined directory depending on your OS:
+Unzip GlassFish 7 on the predefined directory depending on your OS:
 
-* Windows: `c:\glassfish6`
-* Unix: `$HOME/glassfish6`
+* Windows: `%HOMEDRIVE%%HOMEPATH%\glassfish7`
```
[the remark this answers is not in the document]
Comment (BalusC) [reply tutorial/README.md:11]: Okay.

File: tutorial/pom.xml [hunk 2]
```diff
@@ -162,7 +162,7 @@
                                     <version>3.2.3</version>
                                 </requireMavenVersion>
                                 <requireJavaVersion>
-                                    <version>1.8</version>
+                                    <version>[11,)</version>
```
[the remark this answers is not in the document]
Comment (BalusC) [reply tutorial/pom.xml:165]: Not understanding. Please elaborate (with an example).

File: tutorial/pom.xml [hunk 3]
```diff
@@ -191,8 +191,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>11</source>
```
[the remark this answers is not in the document]
Comment (BalusC) [reply tutorial/pom.xml:194]: Not understanding. It's already 11. Please elaborate.

File: tutorial/pom.xml [hunk 4]
```diff
@@ -64,7 +64,7 @@
         <eclipselink.version>3.0.0</eclipselink.version>
         <glassfish.embedded.version>6.0.0</glassfish.embedded.version>
 
-        <glassfish.home>${session.executionRootDirectory}/target/glassfish7</glassfish.home>
+        <glassfish.home>${user.home}/glassfish7</glassfish.home>
```
[the remark this answers is not in the document]
Comment (BalusC) [reply tutorial/pom.xml:67]: Okay, adjusted, but it didn't work for me. I'll need to look closer at that later.

**Pull request merged**